### ISSUE TITLE
Create and publish app

### ISSUE THREAD

**Issue by Octogonapus** [domluna/JuliaFormatter.jl#580]
Related to https://github.com/domluna/JuliaFormatter.jl/issues/498, we'd like to use this package to format our Julia projects using a pre-commit hook. We wouldn't want to wait for the Time To First Format during each commit, though. I propose that an app is created (and published in this repo's releases) using PackageCompiler.jl. An app would allow the user to run the formatter from the cmdline like `./JuliaFormatter path/to/my/code`. It would also meet our latency requirements, as all the formatting code would already be compiled. I've made a proof of concept of this locally and it gives me confidence that this would work well.

I'm happy to implement this, but my question to you is, do you want this feature? If you do, I'll work on PRs for the changes to this repo.

**Comment by domluna** [comment]
that sounds great! I haven't dabbled with PackageCompiler before so the help would be very much welcomed

**Comment by musoke** [comment]
This may be a duplicate of  #483

**Comment by domluna** [comment]
> This may be a duplicate of #483

Once this is done it would close #483

**Comment by jayvdb** [comment]
I think this would also close https://github.com/domluna/JuliaFormatter.jl/issues/105

**Comment by penelopeysm** [comment]
Closed by #947
